tests/dummy/app/templates/docs/guides/upgrading.md

```diff
@@ -6,7 +6,7 @@ While upgrading from 2.x to 3.x should not be very time-consuming in most cases,
 
 ```sh
 yarn-or-npm remove ember-electron
-ember install ember-electron
+ember install ember-electron@3
```

Comment:
Should we maybe do `@beta` instead?

Reply:
Well, then if we come out with `4.0.0-beta.1` later it will be wrong. Doesn't `@3` work? (I didn't actually test)

Reply:
I don't think we would keep the instructions once a new beta was out. They replaced the old 1.x to 2.x upgrade instructions. We could split them out and have separate upgrading pages for 1 -> 2, and 2-> 3.